src/uct/api/uct_def.h

```diff
@@ -15,9 +15,9 @@
 #include <sys/types.h>
 
 
-#define UCT_COMPONENT_NAME_MAX     16
+#define UCT_COMPONENT_NAME_MAX     32
 #define UCT_TL_NAME_MAX            10
-#define UCT_MD_NAME_MAX            16
```

Comment:
memory domain name does not include "ib/" prefix anymore.
is this still needed?

Reply:
i think that the change of UCT_MD_NAME_MAX may be needed.
when I use ucx_info -d, it shows that 
![image](https://user-images.githubusercontent.com/50832474/67665678-2e1bba80-f9a5-11e9-9782-e6dd7f918b06.png)

Reply:
@shenyang-3390 are you using latest master branch?

Reply:
the version I am using is 1.6.0
the latest master is ok. could you update it for 1.6?

Reply:
i'm afraid that for v1.6 it's too big change which includes also API change
I'd suggest to use https://github.com/openucx/ucx/releases/tag/v1.7.0-rc1 which also has this, and going to be released soon

Reply:
ok ,3ks